lib/buttons-and-links/KExternalLink.vue

```diff
@@ -78,18 +66,18 @@
     },
     computed: {
       /**
-       * If icon is provided, add 8px margin between the icon and the text
+       * If link opens in new tab, add 8px margin between the icon and the text
        */
       spanStyle() {
-        if (this.icon) {
```

Comment:
This bit of the logic needs to be applied to your changes to avoid the margin showing when we don't have an icon. During testing I see this on the Usage & Privacy on Kolibri (you can find the Usage & Privacy modal clicking the link at the bottom of the main sidebar).

![image](https://user-images.githubusercontent.com/6356129/103575609-d6b98a00-4e86-11eb-8c26-fa6129112157.png)